ai/langchain4j/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>langchain4j</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: AI LangChain4J</name>
+    <properties>
+        <!-- Langchain4j extensions are not part of Quarkus core -->
+        <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+        <skipTests>true</skipTests>
+        <skipITs>true</skipITs>
+    </properties>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>${quarkus.platform.group-id}</groupId>
+                <artifactId>quarkus-langchain4j-bom</artifactId>
```

Comment:
I am not trying to be annoying, but are you certain that if this BOM is removed, it has any impact on test results? Can't test it as I don't have a key (or money :grinning: ). The reason why I ask - it influences if I accept your explanation regarding `quarkus.platform.group-id`. Please try it.